lib/keybase/local/config.rb

```diff
@@ -9,6 +9,8 @@ module Config
       # The Keybase configuration directory.
       CONFIG_DIR = if Gem.win_platform?
                      File.expand_path("#{ENV["LOCALAPPDATA"]}/Keybase").freeze
+                   elsif /darwin/.match?(RUBY_PLATFORM)
```

Comment:
Let's use `=~` here instead of `match?` (like #3), since `match?` apparently wasn't added until 2.4.

Other than that, looks good!

Reply:
I wanted to be consistent with the match at the bottom. Fixed.